content/departments/security/tooling/entitle.md

```diff
@@ -0,0 +1,30 @@
+# Entitle
+
+## What is it?
+
+Entitle is an authentication and access management system. It is a cloud based application that automates access requests, approvals, and revocation of permissions based on custom policies. All of this can be done via slack with a simple command “/entitle”.
```

Comment:
```suggestion
Entitle is an authentication and access management system. It is a cloud based application that automates access requests, approvals, and revocation of permissions based on custom policies. All of this can be done via Slack with a simple command “/entitle”.
```

nit